cmake/templates/ngraphConfig.cmake.in

```diff
@@ -28,11 +28,11 @@
 #
 #   ngraph::frontend_manager               - nGraph frontend manager
 #
-#   ngraph_onnx_ngraph_frontend_FOUND      - True if the system has onnx_ngraph_frontend library
-#   ngraph::onnx_ngraph_frontend           - ONNX FrontEnd target (optional)
+#   ngraph_onnx_ov_frontend_FOUND          - True if the system has onnx_ov_frontend library
```

Comment:
I think for the backward compatibility we shouldn't rename variables. (it is obsolete cmake config)

Reply:
Not applicable to this PR anymore